bagxml/README.md

```diff
@@ -0,0 +1,85 @@
+Bag XML Schema
+==============
+
+
+Work-In-Progress
+----------------
+
+This readme, the schema, and the example document are all still a
+work-in-progress. Major changes to all the three documents are expected
+to take place.
+
+
+About
+-----
+
+This schema defines what the UNT Libraries considers to be a valid bagxml
+instance. A bagxml instance is an XML representation of a bag.
```

Comment:
We should mention that the bag that this is a representation of is a bag as defined by BagIt specification.